File: java/01-setup-hello-world/src/test/java/org/first/tutorial/ExerciseTest.java
package org.first.tutorial;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class ExerciseTest {

    @Test
    void greetsByName() {
        Exercise exercise = new Exercise();
        assertEquals("Hello, Driver! Ready to boot the robot brain.",
                exercise.greet("Driver"));
    }

    @Test
    void trimsWhitespace() {
        Exercise exercise = new Exercise();
        assertEquals("Hello, Pilot! Ready to boot the robot brain.",
                exercise.greet("  Pilot  "));
    }
}

File: kotlin/01-setup-hello-world/src/test/kotlin/org/first/tutorial/ExerciseTest.kt
package org.first.tutorial

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class ExerciseTest {

    @Test
    fun greetsByName() {
        val exercise = Exercise()
        assertEquals("Hello, Driver! Ready to boot the robot brain.", exercise.greet("Driver"))
    }

    @Test
    fun trimsWhitespace() {
        val exercise = Exercise()
        assertEquals("Hello, Pilot! Ready to boot the robot brain.", exercise.greet("  Pilot  "))
    }
}
